src/battle.ts

```diff
@@ -2966,14 +2973,14 @@ class Battle {
 		pokemonid = parsedPokemonid;
 
 		const searchid = `${pokemonid}|${details}`;
-		const side = this.sides[siden];
+		const side = this.gameType === 'multi' ? this.getSide(`p${siden + 1}`) : this.sides[siden];
```

Comment:
Why is `getSide` required here? I can't think of a reason why `this.sides[siden]` wouldn't work for everyone, unless something's seriously wrong.